examples/readers/pytorch_data_api_tiledb_dense.ipynb

```diff
@@ -387,7 +391,11 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```

Comment:
Generally you might need to re-run them I see that are many leftovers here and there in the notebooks.